src/Parser/BrewmasterMonk/Modules/Features/AlwaysBeCasting.js

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import SPELLS from 'common/SPELLS';
+import Icon from 'common/Icon';
+import { formatPercentage } from 'common/format';
+import StatisticBox, { STATISTIC_ORDER } from 'Main/StatisticBox';
+import CoreAlwaysBeCastingHealing from 'Parser/Core/Modules/AlwaysBeCastingHealing';
+
+class AlwaysBeCasting extends CoreAlwaysBeCastingHealing {
+  static ABILITIES_ON_GCD = [
+    SPELLS.CHI_TORPEDO_TALENT.id,
+    SPELLS.DETOX.id,
+    SPELLS.LEG_SWEEP_TALENT.id,
+    SPELLS.PARALYSIS.id,
+    SPELLS.ROLL.id,
+    SPELLS.CRACKLING_JADE_LIGHTNING.id,
+    SPELLS.TRANSCENDENCE.id,
+    SPELLS.TRANSCENDENCE_TRANSFER.id,
+    SPELLS.TIGER_PALM.id,
+    SPELLS.BLACKOUT_STRIKE.id,
+    SPELLS.KEG_SMASH.id,
+    SPELLS.BREATH_OF_FIRE.id,
+    SPELLS.EXPLODING_KEG.id,
+    SPELLS.RUSHING_JADE_WIND.id,
+    SPELLS.EXPEL_HARM.id,
+    SPELLS.INVOKE_NIUZAO_THE_BLACK_OX_TALENT.id,
+    SPELLS.RING_OF_PEACE_TALENT.id,
+    SPELLS.SUMMON_BLACK_OX_STATUE_TALENT.id,
+  ];
+
+  // BrM has a fixed 1s GCD 
+  static calculateGlobalCooldown(haste) {
+    return 1;
+  }
+
+  suggestions(when) {
+    const deadTimePercentage = this.totalTimeWasted / this.owner.fightDuration;
+
+    when(deadTimePercentage).isGreaterThan(0.4)
+      .addSuggestion((suggest, actual, recommended) => {
+        return suggest('Your dead GCD time can be improved. Try to Always Be Casting (ABC); try to reduce the delay between casting spells and when you\'re not healing try to contribute some damage.')
+          .icon('spell_mage_altertime')
+          .actual(`${formatPercentage(deadTimePercentage)}% dead GCD time`)
+          .recommended(`<${formatPercentage(recommended)}% is recommended`)
+          .regular(recommended + 0.15).major(1);
```

Comment:
This seems like a large jump in threshold; I think doing nothing for a long time as a brewmaster is detrimental as (iirc) you should continue to build ISB time when you're not actually tanking.